_rules/auto-update-text-efbfc7.md

```diff
@@ -30,9 +30,9 @@ The rule applies to any [HTML element][] that has a [visible][] [text node][] as
 
 **Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Content that updates less frequently, or that doesn't update on a regular interval, will be inapplicable for this rule but may nonetheless fail [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. It is however a much lesser accessibility issue, and without any time limit this rule would be virtually untestable.
 
-## Expectation
+## Expectation 1
 
-For each test target there exists an [instrument][] in the same [web page](#web-page-html), or in another [web page](#web-page-html) that is [linked][hyperlink] from the [web page](#web-page-html) of the test target, to achieve one of the following objectives:
+For each test target there exists an [instrument][] to achieve one of the following objectives:
```

Comment:
I think we need to go the "set of instruments" path here also.
There could be one button for pausing and a different one for resuming.

Reply:
I agree. The new example makes that clear :-)